src/main/resources/dockerfiles/Dockerfile-skeleton

```diff
@@ -0,0 +1,42 @@
+ARG BASE_IMAGE=xeniteu/tomcat:alfresco-6.0-ubuntu
+FROM $BASE_IMAGE
+
+ARG SHARE_VERSION_MAJOR
+ARG SHARE_VERSION_MINOR
+ARG SHARE_VERSION_REV
+ARG SHARE_FLAVOR
+
+# ${CATALINA_HOME} is set to ${CATALINA_HOME} in upstream
+
+# download location alf
+ENV SHARE_VERSION ${SHARE_VERSION_MAJOR}.${SHARE_VERSION_MINOR}.${SHARE_VERSION_REV}
```

Comment:
We don't need to construct these environment variables anymore, since we don't download share inside the Dockerfile

Reply:
True, but it's nice to have it as an environment variable inside the container imo. 
But - it sohuld be overwritten by the real version of community or enterprise.